Pegasus/include/Geometry.hpp

```diff
@@ -157,26 +165,23 @@ class Box : public SimpleShape
 namespace intersection
 {
 // Utility functions
-bool IsSameSidePoint(
-    glm::dvec3 const& p1, glm::dvec3 const& p2, glm::dvec3 const& a, glm::dvec3 const& b);
-
 template <typename VerticesContainerIt>
 void CalculateBoxVertices(
     glm::dvec3 const& i, glm::dvec3 const& j, glm::dvec3 const& k, VerticesContainerIt verticesIterator)
 {
     *verticesIterator = (i + j + k);
-    *(verticesIterator + 1) = ( i - j + k);
-    *(verticesIterator + 2) = ( j - i + k);
+    *(verticesIterator + 1) = (i - j + k);
```

Comment:
Maybe small 2 line comment how calculation is working here? Cause if someone will support this code, will take a time to understand whats going on here